README.md

```diff
@@ -1,3 +1,11 @@
+
+Image Credit:
+Blanket: My own
+Scarf: unsplash.com
+Embroidery: unsplash.com
+
```

Comment:
Sorry, I know that this is a weird place for this comment, but your json db is on port 3000 and your react app is also on 3000, when you start the react app after the json db it asks if you want to use another port but I would change that in your code

Reply:
Oh, good point!  I'll have to change that